test_runner/regress/test_logical_replication.py

```diff
@@ -203,6 +203,75 @@ def slot_removed(ep):
     wait_until(number_of_iterations=10, interval=2, func=partial(slot_removed, endpoint))
 
 
+# Tests that walsender correctly blocks until WAL is downloaded from safekeepers
+def test_lr_with_slow_safekeeper(neon_simple_env: NeonEnv, vanilla_pg):
+    env = neon_simple_env
+
```

Comment:
This test runs with a single safekeeper by default, you need to add this to use 3 safekeepers:
```
neon_env_builder.num_safekeepers = 3
```

Reply:
Done